portal/src/graphql/portal/OAuthClientConfigurationScreen.tsx

```diff
@@ -63,10 +84,54 @@ function makeOAuthClientListColumns(
   ];
 }
 
+const ConfirmRemoveOAuthClientDialog: React.FC<ConfirmRemoveOAuthClientDialogProps> = function ConfirmRemoveOAuthClientDialog(
+  props: ConfirmRemoveOAuthClientDialogProps
+) {
+  const {
+    visible,
+    updatingAppConfig,
+    onDismiss,
+    removeOAuthClient,
+    clientId,
+    clientName,
+  } = props;
+  const { renderToString } = useContext(Context);
+
+  const onConfirm = useCallback(() => {
+    removeOAuthClient(clientId);
+    onDismiss();
```

Comment:
We should keep the dialog open while the API request is in progress, similar to other dialogs. (e.g. disconnect identity)